0. Kirish/0.3 Guvohliklar.md

```diff
@@ -0,0 +1,207 @@
+
+Sharaf ElDean Abd-ElKhalik Mansour
+Dasturiy Muhandis
+Qohira universiteti, Misr
+
+---
+
+"Me'yor menga harqanday tizimni loyihalashda boshqacha o'ylashimga yordam berdi."
+
+Awais Shabir
+Yetakchi Dasturiy Muhandis
+Cloud Solution, Saudia Arabistoni
+
+---
+
+"Me'yor men boshqaradigan jamoalarga umumiy arxitektura va e'tiborini yondashuvga qaratadi."
+
+Dennis Landi
+Texnik Arxitektor
+Allied Data, Inc., AQSH
+
+---
+
+"Me'yor haqiqiy dasturiy ta'minot muhandisining ta'sirini amalga oshirish uchun to'liq imkoniyatlarni beradi, o'zlarini, sifat jihatidan, sifatli va davomiy bo'lgan dasturiy ta'minotini etkazib beradi."
+
+Shri Humrudha Jagathisun
+Katta dasturiy ta'minot muhandisi
+Microsoft, AQSH
+
+---
+
+"Me'yor xavotirni uzoqlashtirishga yordam beradi. Turlixil jamoalar o'rtasidagi loyiha qoliblarini birlashtirishga yordam beradi."
+
+Raúl Lorenzo Boulllosa
+Dasturchi Muhandis
+Altia Consultores, Ispaniya
+
+---
+
+"Me'yor menga chiroyli kod yozishda yordam berdi. Testga asoslangan yechimlarni yozish va mening jamoam samarali ishlashni boshladi."
+
+Mabrouk Mahdhi
+Katta dasturiy ta'minot muhandisi
+Messer SE & Co. KGaA, Germaniya
+
+---
+
+"Me'yor turli tajribaga ega dasturchilar tomonidan dasturni amalga oshirishda yordam beradi. TDDni to'g'ri qo'llash. Soddalashtirish. Kod sifatini ta'minlash uchun kod sharhlarida aniq va foydalanilgan me'yorni belgilang."
+
+Christo du Toit
+Katta raqamli Dasturchi
+Mahalliy Tibbiy Xizmat, Angliya
+
+---
+
+"Me'yor oson dizaynlashda yordam berdi, kod asoslarini o'sh tez va samarali, vazifalarni rejalashtirish yengil"
+
+Kailu Hu
+Dasturchi Muhandis
+Microsoft, AQSH
+
+---
+
+"Me'yor Dasturlarni ishlab chiqish tezligini oshirishga yordam berdi."
+
+Yusuf Corr
+Dasturchi muhandis
+Microsoft, AQSH
+
+---
+
+"Me'yor jamoamizga kod sifatini amalga oshirishimizga yordam berdi va shu bilan birgalikda muvafaqqiyatsizliklarni oldini oldi. Bundan tashqari murrakkab bo'lmagan dasturlarni ishlab chiqishga yordam berdi va xatolarni kamaytirdi."
+
+Marthin Thomas
+Dasturchi Muhandis
+Alpha Events and Marketing, Namibia
+
+---
+
+"Me'yordan bilimlarimni mustahkamlab web dasturchi sifatida ishga kirdim."
+
+Florian Renard
+Web dasturchi
+HR Team, Fransiya
+
+---
+
+"Yaxshiroq ichki va jamoaviy hamkorlik. Kamroq meetinglar va bo'shvaqtlarni samarali sarflash. Birqancha umumiy tushunchalar bundan tashqari aloqa va hamkorlikni rivojlantiradigan so'zlar."
```

Comment:
PRs, continuous integration/feedback and engineering quality become automatic. Easily one of the best ways to remove the roadblocks that inhibit new teams." qolgan